apps/meteor/client/views/room/providers/RoomProvider.tsx

```diff
@@ -27,6 +28,16 @@ const RoomProvider = ({ rid, children }: RoomProviderProps): ReactElement => {
 
 	const roomQuery = useReactiveQuery(['rooms', rid], ({ rooms }) => rooms.findOne({ _id: rid }));
 
+	const subscribeToRoom = useStream('room-data');
```

Comment:
Let's move this to the OmnichannelProvider instead

Reply:
Unfortunately we don't have room context there :/